src/Panel.tsx

```diff
@@ -146,7 +146,7 @@ export const Panel = ({ active, api }: PanelProps) => {
         <VisualTests
           dismissBuildError={() => setLocalBuildProgress(undefined)}
           localBuildProgress={localBuildIsRightBranch ? localBuildProgress : undefined}
-          startDevBuild={() => emit(START_BUILD)}
```

Comment:
Are we worried about emitting the access token over the channel unencrypted?

A malicious addon could watch for it easily enough. I think this is the full user token (not just a stripped down token that only has the ability to run builds ~= the project token), right?

Reply:
I asked this question [in Slack](https://chromaticqa.slack.com/archives/C02K88CKTFW/p1699269065837069). The token is already exposed to malicious addons, because it's in localStorage. Generally it's up to the user's own responsibility not to install untrusted addons.

And yes this is the user's OAuth token for the addon, not an app token (which is used to run builds).

Reply:
I guess there is no way to encrypt it that couldn't be decrypted by an addon that has access to our source code.

Reply:
It's all client side, so no.